README.md

```diff
@@ -268,16 +270,16 @@ end
 ```
 
 To use the `has_secure_password` macro, you'll need to add `gem 'bcrypt'` to
-your Gemfile if it isn't already.
+your Gemfile if it isn't there already.
 
 When using [`has_secure_password`][has_secure_password], Rails will use the
 `bcrypt` gem to hash and salt all passwords on the `User` model.
 
-[`has_secure_password`][has_secure_password] adds two fields to your model:
-`password` and `password_confirmation`. These fields don't correspond to
-database columns! Instead, the method expects there to be a `password_digest`
-column defined in your migrations. To make this work, your `users` table
-**must** have a `password_digest` column:
+The [`has_secure_password`][has_secure_password] method will also recognize and
+automatically handle two fields in your form: `password` and
+`password_confirmation`. These fields don't correspond to database columns!
+Instead, to make this work, your `users` table **must** have a `password_digest`
+column:
```

Comment:
`in your form` isn't right in this context, but `fields` is also not quite right. How about this:

The [`has_secure_password`][has_secure_password] method also provides two new instance methods on your `User` model: `password` and `password_confirmation`. These methods don't correspond to database columns! Instead, to make these methods work, your `users` table **must** have a `password_digest` column: